packages/react-native/template/ios/Project.swift

```diff
@@ -0,0 +1,117 @@
+import ProjectDescription
+
+let reactNativeXCodeScript =
+"""
+set -e
+WITH_ENVIRONMENT=\"../node_modules/react-native/scripts/xcode/with-environment.sh\"
+REACT_NATIVE_XCODE=\"../node_modules/react-native/scripts/react-native-xcode.sh\"
+/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"
+"""
+
+let startPackagerScript =
+"""
+export RCT_METRO_PORT=\"${RCT_METRO_PORT:=8081}\"
```

Comment:
Should we consider putting this into a script file to make it easier to edit? Or do we rather want to keep the Xcode project as close as possible to what it used to be?

Reply:
The goal will be to remove these scripts and have them in separate files.
However, I didn't want to push too many changes in a single sweep... 😅